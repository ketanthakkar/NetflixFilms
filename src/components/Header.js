import React from 'react';
import Title from './Title';
import Search from './Search';

export const Header = () => (
        <header className="header-section">
            <Title />
            <Search />
        </header>
);

export default Header;
