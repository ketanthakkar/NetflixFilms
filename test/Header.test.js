import React from 'react';
import { shallow } from 'enzyme';
import Header from '../src/components/Header';

describe('should render Header correctly', () => {
  const tree = shallow(
      <Header />,
  );
  test('Snapshot test', () => {
    expect(tree).toMatchSnapshot();
  });
});
