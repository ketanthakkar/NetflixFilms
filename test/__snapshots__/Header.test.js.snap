// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly Snapshot test 1`] = `
<styled.header>
  <Title />
  <Connect(Search) />
</styled.header>
`;
